sklearn/tests/test_multiclass.py

```diff
@@ -48,6 +49,12 @@ def test_ovr_exceptions():
                   np.array([[1.5, 2.4], [3.1, 0.8]]))
 
 
+def test_check_classification_targets():
+    # Test that check_classification_target return correct type. #5782
+    y = np.array([0.0, 1.1, 2.0, 3.0])
+    assert_raises(ValueError, check_classification_targets, y)
```

Comment:
This would pass previously as well. Can you use `assert_warns`_message` and check that y_type is present in the message?